get_ioc_usage.py

```diff
@@ -39,7 +41,7 @@ def print_iocs_on_instruments(instrument_configs):
     :param instrument_configs: instrument ioc config dictionary
     """
 
-    for instrument, iocs in instrument_configs.items():
+    for instrument, iocs in list(instrument_configs.items()):
```

Comment:
I think the `futurize` module has been overly cautious here by casting to a list. In this case it would be better to just call `instrument_configs.items()` rather than `list(instrument_configs.items())`

Please apply this change throughout the code where unnecessary casts to `list` have been added

Reply:
Yeah, I suspected those extra casts might have been unnecessary. I've removed them in the last commit